backend/requirements.txt

```diff
@@ -14,3 +14,5 @@ requests==2.28.1
 types-paramiko==2.12.0.1
 types-requests==2.28.11.5
 uvicorn==0.20.0
+PyPDF2==3.0.1
+python-multipart==0.0.6
```

Comment:
is this import needed / used?

Reply:
do u mean are the requirements necessary? yes

Reply:
okay in that case, looks good to merge